.github/workflows/tests.yml

```diff
@@ -61,17 +63,22 @@ jobs:
         run: vendor/bin/pest
         env:
           DB_CONNECTION: mysql
-          DB_USERNAME: root
+          DB_DATABASE: pulse
+          DB_USERNAME: pulse
+          DB_PASSWORD: password
+          DB_COLLATION: utf8mb4_unicode_ci
```

Comment:
Setting the credentials explicitly instead of relying on the defaults which recently changed.

The collation needs to be explicitly included here because the default collation is not supported on MySQL 5.7 where these tests are running.